zilliqa/src/contracts/deposit.sol

```diff
@@ -68,4 +72,12 @@ contract Deposit {
     function getStakers() public view returns (bytes[] memory) {
         return _stakerKeys;
     }
+
+    function getPeerId(bytes calldata blsPubKey) public view returns (bytes memory) {
```

Comment:
Maybe an optimisation for the future, but potentially we should have a single method that returns `(stake, rewardAddress, peerId)` to limit the number of EVM calls we need to make?